### PR TITLE
Add graphs disaggregated by gender and a pie of gender; switch to plotly.

### PR DESCRIPTION
**Pull request by as2388**
Adds 3 new types of graph:
 - A graph showing the distribution of normal codes for RQA, disaggregated by gender:
    - as absolute numbers
    - as fractions of the number of people who mentioned that theme out of all the people with that gender.
 - A pie chart of gender.

I needed to switch away from altair to do this, because it couldn't support all of the above, so the new graphs are now being rendered with plotly. Plotly's API is easier to use and more capable, so the graphing code is now a little shorter and more readable, especially for the new graphs.

This PR also migrates the existing RQA distribution graph from altair to plotly to show what needs to change to convert existing graphs to use the new library. The remaining graphs will be migrated and the dependency on altair removed in a future PR.

Examples of all the new/changed graphs can be found here: https://drive.google.com/drive/folders/1ftPHrliUk1LpaMtVY4k7wXgLiaI4FAyK?usp=sharing

### CODE REVIEW

File: generate_analysis_graphs.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from glob import glob
 
 import altair
```
Comment (lukechurch) [inline generate_analysis_graphs.py:8]: Is this still used?

**Review by lukechurch — APPROVED** [review]
Thanks @as2388

File: generate_analysis_graphs.py (hunk 1 above)
Comment (as2388) [reply generate_analysis_graphs.py:8]: Yes, because this only a partial migration. It's removed completely in #30

**Comment by as2388** [comment]
cc @IsaackMwenda. Once the work on COVID19 has calmed down and we're back to thinking about automated analysis for other projects, we should migrate those away from altair too.

**Pull request merged**